crates/deltalake-sql/src/parser.rs

```diff
@@ -138,10 +138,6 @@ impl<'a> DeltaParser<'a> {
         match self.parser.peek_token().token {
             Token::Word(w) => {
                 match w.keyword {
-                    Keyword::DESCRIBE => {
```

Comment:
DataFusion relies on `sqlparser`, which handles most of the parsing. As of the new version used in DataFusion 35, it also handles parsing DESCRIBE statements. DataFusion removed their custom DescribeTableStmt as a result, and I had to remove it here as well to match.